apps/linear_algebra/benchmarks/CMakeLists.txt

```diff
@@ -34,19 +34,25 @@ target_compile_definitions(halide_benchmarks PRIVATE -DUSE_HALIDE ${MISC_DEFINIT
 target_link_libraries(halide_benchmarks PRIVATE halide_blas)
 list(APPEND BLAS_NAMES halide)
 
-find_package(Eigen3 QUIET)
-if (NOT Eigen3_FOUND)
+find_package(PkgConfig)
+if (PkgConfig-NOTFOUND)
+  find_package(Eigen3 QUIET)
+  set(EIGEN3_FOUND NOT Eigen3-NOTFOUND)
+  if (EIGEN3_FOUND)
+    set(EIGEN3_LDFLAGS Eigen3::Eigen)
+  endif()
```

Comment:
Prefer `if (TARGET Eigen3::Eigen)` ... no need for this `set()` call.

Reply:
Also just call the variable `EIGEN3` or maybe `EIGEN3_TARGET`, but not `LDFLAGS`

Reply:
I only called it _LDFLAGS for consistency with pkgconfig, but that's gone now.